Feature/93

SimoPP opened the pull request:
Created the guess_imap() and guess_smtp() functions in app.py. 

They connect to the target server trying to find the port and protocol settings.
They return a dictionary with the settings

SimoPP committed:
written guess_imap() function

SimoPP committed:
written guess_smtp() function and updated docstrings

MattiaFailla commented on app.py:
```diff
@@ -393,98 +394,88 @@ def get_email_platform():
+                ssl_context=None
+            )
+            test = connection.connection.check()
+            print(test)
```
Please use Tentalog

MattiaFailla commented on app.py:
```diff
@@ -393,98 +394,88 @@ def get_email_platform():
+                ssl_context=None
+            )
+            test = connection.connection.check()
+            print(test)
```
Please use Tentalog

SimoPP replied:
All corrected, thanks!

MattiaFailla reviewed (commented):
Please add a delay between attempts to prevent the server from adding the client to the blocked ones. In addition, it would be better to migrate to Tentalog than to use the print statement.

SimoPP committed:
Corrected guess_smtp() function

deleted starttls, not supported by SMTP_SSL
Added a delay between the attempts to connect to the server.
Used the logger for the debug!

MattiaFailla commented:
![image](https://user-images.githubusercontent.com/11872425/80866137-7934c400-8c8d-11ea-9bf6-18a60bd4d336.png)

MattiaFailla committed:
Ignoring log

SimoPP committed:
Corrected the guilty variable.

SimoPP committed:
Merge remote-tracking branch 'origin/feature/93' into feature/93
